test/e2e/LangsScenario.spec.ts

```diff
@@ -0,0 +1,72 @@
+import app from '../../src/server';
+import DB from '../../src/models';
+import * as utils from "../utils/utils";
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+chai.use(chaiHttp);
+const {expect} = chai;
+
+const APIKEY = '7718330d2794406c980bdbded6c9dc1d';
+
+describe('GET api/langs', () => {
+    before(async () => {
+        await DB.langs.bulkCreate([
+            {lang_slug: 'py2', lang_name: 'Python 2', lang_version: '2.7'},
+            {lang_slug: 'py3', lang_name: 'Python 3', lang_version: '3.6'},
+            {lang_slug: 'java8', lang_name: 'Java', lang_version: '1.8'}
+        ]);
+        await DB.apikeys.create({
+            id: 1,
+            key: APIKEY,
+            whitelist_domains: ['*'],
+            whitelist_ips: ['*']
+        });
+    });
+    after(utils.truncateTables);
+
+
+    it('should throw 403 error API key is absent in the request', async () => {
+        const res = await chai.request(app).get(`/api/langs`);
+        expect(res.status).to.equal(403);
+        // TODO this should be res.body.err maybe? (consistent with other error)
```

Comment:
Status code will define the error and let's stick with body.message

Reply:
But the base validator is sending `body.err`  [base-validator-error](https://github.com/coding-blocks/judge-api/blob/72944d1764f56a6c06af7b1a46c90cfd16872e7d/src/validators/baseValidator.ts#L16)